src/pages/SignupPage.js

```diff
@@ -83,9 +83,22 @@ const SignupPage = () => {
             onChange={(e) => setUsername(e.target.value)}
             error={usernameError}
             disabled={isLoading}
+            autoComplete='true'
+            autoFocuss
```

Comment:
I think its a typo, `autoFocuss` should be `autoFocus`

Reply:
ops! Ill fix that.